src/patterns/dosdp-patterns/bone_length_from_to.yaml

```diff
@@ -0,0 +1,86 @@
+---
+pattern_name: bone_length_from_to
```

Comment:
does this belong in the PR?

Reply:
No. It should be in another github branch. I am not sure why it shows up here.

Reply:
I have deleted the file from this github branch/